recipes/minimac4/build.sh

```
@@ -0,0 +1,41 @@
+
+# Set up dependencies and directories
+cget ignore xz
+cget install -f ./requirements.txt
+cmake --version
+mkdir -p build
+cd build
+
+# Set CXXFLAGS to include the correct path to lzma.h and LDFLAGS to include the path to liblzma
+export CXXFLAGS="-I${PREFIX}/include -I${PREFIX}/bin -I${PREFIX}/lib"
+export LDFLAGS="-L${PREFIX}/lib -llzma"
+export CMAKE_MAKE_PROGRAM=$(which make)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)